cookbooks/aws-parallelcluster-common/resources/lustre/lustre_redhat8.rb

```diff
@@ -29,18 +29,32 @@
     log "FSx for Lustre is not supported in this RHEL version #{version}, supported versions are >= 8.2" do
       level :warn
     end
-  elsif node['cluster']['kernel_release'].include? "4.18.0-425.3.1.el8"
-    log "FSx for Lustre is not supported in kernel version 4.18.0-425.3.1.el8 of RHEL, please update the kernel version" do
-      level :warn
-    end
   else
     action_install_lustre
   end
 end
 
+def find_os_minor_version
```

Comment:
can we put this function in common for both rhel and centos? the versioning of the kernel is actual the same